website/project/views/contributor.py

```diff
@@ -378,6 +378,55 @@ def project_manage_contributors(auth, node, **kwargs):
     return {}
 
 
+@no_auto_transaction
+@must_be_valid_project  # injects project
+@must_have_permission(ADMIN)
+@must_not_be_registration
+def project_remove_contributor(auth, **kwargs):
```

Comment:
@billyhunt Looks like in a8286ca52b65ee8e959078fe41ac940d44423564 you decided to fold the old `project_removecontributor` into the new `project_remove_contributor`, yes? I'll update the [punchlist](https://github.com/CenterForOpenScience/osf.io/pull/4799#issuecomment-170572129) accordingly.